From 8ed54d68ad25f469985d3ed4e74c9fd007f3d372 Mon Sep 17 00:00:00 2001
From: Friedrich Tschirpke <friedrich.tschirpke@live.de>
Date: Sat, 8 Jun 2024 15:12:53 +0200
Subject: [PATCH] feat: remove wireless and battery from i3status

---
 i3status/config | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/i3status/config b/i3status/config
index ec168da..030fd1a 100644
--- a/i3status/config
+++ b/i3status/config
@@ -11,28 +11,17 @@ general {
         interval = 5
 }
 
-order += "wireless _first_"
 order += "ethernet _first_"
-order += "battery all"
 order += "disk /"
 order += "load"
 order += "memory"
 order += "tztime local"
 
-wireless _first_ {
-        format_up = "W: (%quality at %essid) %ip"
-        format_down = "W: down"
-}
-
 ethernet _first_ {
         format_up = "E: %ip (%speed)"
         format_down = "E: down"
 }
 
-battery all {
-        format = "%status %percentage %remaining"
-}
-
 disk "/" {
         format = "%avail"
 }
-- 
2.45.2

